techmatters/flex-plugins
# CHI-2573 Add "Skills" column to table in Teams View

## Description
- This PR adds a new column for displaying skills for each agent/counselor on the Teams View page. 
- The cell caters to the display of up to three skills. If one is assigned more than three skills, an expandable button displaying '... see more' captioned(accessibility for screen readers) "see more skills for <counselor’s name>" is provided. Upon interaction, this button reveals all the hidden skills. Subsequently, '... see less' toggles to collapse
- Much of CategoryWithToolTip logic is used in SkillWithToolTip with some changes. Hence, the related dependencies have been moved to a global state. 
- In terms of styling, 'PillBase' is created to unify similar components across the codebase(note that status in profiles and category pills are using a similar styling). 

### Checklist
- [x] [Corresponding issue has been opened](https://tech-matters.atlassian.net/browse/CHI-2573)
- [n/a] Feature flags added
- [n/a] Tested for chat contacts
- [n/a] Tested for call contacts

### Other Related Issues

None

### Verification steps
- Run Flex, be available online, and go to the Teams View page.
- Check that the acceptance criteria in jira ticket are fulfilled
